commit 5c5b191b27e19927d67ffcde88adaaf4b47a2d53
Author: AndDiSa <anddisa@googlemail.com>
Date:   Tue Apr 25 08:45:27 2017 +0200

    improve compatibility when compiling with clang
    
    when compiling with USE_CLANG_PLATFORM_BUILD := true compilation of hwui
    fails because the optimization parameter '-mapcs' is not known to the
    clang compiler.
    
    Change-Id: Ie99447f47156e58d095953d2b47e2991cbde49c4

diff --git a/libs/hwui/Android.mk b/libs/hwui/Android.mk
index cacfce16ab9a..a38e3266e1f4 100644
--- a/libs/hwui/Android.mk
+++ b/libs/hwui/Android.mk
@@ -149,7 +149,10 @@ endif
 
 ifdef HWUI_COMPILE_FOR_PERF
     # TODO: Non-arm?
-    hwui_cflags += -fno-omit-frame-pointer -marm -mapcs
+    hwui_cflags += -fno-omit-frame-pointer -marm
+ifneq (true, $(USE_CLANG_PLATFORM_BUILD))
+    hwui_cflags += -mapcs
+endif
 endif
 
 # This has to be lazy-resolved because it depends on the LOCAL_MODULE_CLASS
